code/game/objects/structures/trash_pile.dm

```diff
@@ -21,9 +21,9 @@
 		/obj/item/device/perfect_tele,
 		/obj/item/weapon/bluespace_harpoon,
 		/obj/item/weapon/gun/energy/netgun,
-		/obj/item/weapon/moneybag/vault,
+		/obj/item/weapon/gun/projectile/pirate,
 		/obj/item/weapon/permit,
-		/obj/item/weapon/gun/projectile/dartgun
+		/obj/item/clothing/mask/gas/voice
```

Comment:
Voice changer is already on the list, just above the emag.

Reply:
oh derp, well what else should I put?

Reply:
Maybe throw in a /obj/item/device/batterer? It tends to be forgotten and is some sort of weird science thing.